src/components/battle/BattleComponent.vue

```diff
@@ -64,9 +64,10 @@
                     :placeholder="battle.battleOptions.map"
                     @update:model-value="onMapSelected"
                 />
-                <Button>
-                    <Icon :icon="cog" height="23" />
+                <Button @click="openMapList">
+                    <Icon :icon="listIcon" height="23" />
```

Comment:
The previous cog button here was for the map options modal, although I don't think it was working which probably made it confusing. I'll leave this for now and add the cog button back in the future when we've got the map options modal working